client/src/components/SimpleModal.js

```diff
@@ -0,0 +1,116 @@
+import React, { useContext } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Modal from "@material-ui/core/Modal";
+import {
+  MenuItem,
+  Checkbox,
+  ListItemText,
+  FormControl,
+} from "@material-ui/core";
+import { MyContext } from "../Provider";
+import CustomButton from "./CustomButton";
+import API from "../utils/API";
+
+function rand() {
+  return Math.round(Math.random() * 20) - 10;
+}
+
+function getModalStyle() {
+  const top = 50 + rand();
+  const left = 50 + rand();
+
+  return {
+    top: `${top}%`,
+    left: `${left}%`,
```

Comment:
I just got rid of this for now.   I set the top and left at 50% and translate(-50%,-50%). Which is how I have positioned modals before. 
This code came from an example of a modal I was looking at. I had left this in,  meaning to follow up on the random values and why this might be necessary.